### DISCUSSION TITLE
1.1.0

### DISCUSSION THREAD

**Discussion by Carseason** [Carseason/routerdog#21] [Announcements]
### v1.1.0版本功能更新如下：
1. 百万设计师 [tsengwaiyong](https://github.com/tsengwaiyong) 设计了新的图标
2. 新增 桌面下载中心  (需要安装并启用 aria2)
3. 新增 多用户多权限 Webdav 功能 (无依赖)
4. 新增 arp 用户列表, 同时新增上 wol 网络唤醒 (无依赖)
5. 新增定时任务，可以 定时关机重启 
6. 新增快捷关机按钮

**Comment by tsengwaiyong** [comment]
芜湖🎉

**Reply by pandamelive** [reply]
超强
